.changeset/eleven-mugs-rule.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/cms-api": minor
+---
+
+Remove user name from access log interceptor.
+Add request-ip package to identify the users correct ip address.
```

Comment:
Changesets are structured like Git commits. The first line is the header, the rest is the body. So all relevant info should be summed up in the header with more details in the body. Suggestion:

```suggestion
Remove user name from access log and log the correct user IP even if the app is behind a proxy

We decided to remove the user name because of privacy concerns.

The package [request-ip](https://www.npmjs.com/package/request-ip) is now used to get the actual user IP even if the app runs behind a proxy. Previously, the proxy's IP was logged.
```

Reply:
@thomasdax98 we should document this in CONTRIBUTING.md

Reply:
You could also argue that this should be two changesets, one being a patch, and one being a minor change

Reply:
> You could also argue that this should be two changesets

Yes that's also possible

Reply:
Should I split the changeset to two seperate ones?
One with the patch "remove username" and one minor adding the package?

Reply:
@Tobi98 Yes